express-api/src/services/properties/propertiesServices.ts

```diff
@@ -614,7 +616,8 @@ const getPropertiesUnion = async (filter: PropertyUnionFilter) => {
       logger.error('PropertyUnion Service - Invalid Sort Key');
     }
   }
-  return await query.getMany();
+  const properties = await query.getMany();
+  return { properties, totalCount };
```

Comment:
Same idea as with projects.
```suggestion
  const [properties, totalCount] = await query.getManyAndCount();
  return { properties, totalCount };
```

Reply:
Tested using getManyAndCount and it seems to work.